phx/results/admin.py

```diff
@@ -5,11 +5,28 @@
 from .models import Event, Performance, Result
 
 
+class EventInline(admin.StackedInline):
+    """
```

Comment:
Looks a bit like this:

![image](https://github.com/user-attachments/assets/b3821857-d2f4-41f4-9e47-4ad5de7da7c9)